### DISCUSSION TITLE
How do I use the command "input-cmdlist"?

### DISCUSSION THREAD

**Discussion by 1000Fear** [mpv-player/mpv#14390] [Q&A] [answered]
According to reference, `input-cmdlist`prints all commands that can be bound to keys.

What should I do to make it work? I had tried to input it into _mpv.conf_ and _cmd.exe_, but nothing happened.

**Comment by guidocella** [comment]
It's CLI-only.

**Reply by 1000Fear** [reply]
Then how do I get the input command list in windows?

**Comment by kasper93** [comment]
`mpv.com --input-cmdlist`

**Reply by 1000Fear** [reply]
Sorry, I don't quite understand what you mean,

**Reply by kasper93** [reply]
Why do you want to use `input-cmdlist` if you don't understand how to use cli?

**Reply by 1000Fear** [reply]
I want to know all commands that can be bound to keys. Is there any other ways?

**Reply by kasper93** [reply]
To be honest everything can be bound to keys. Depends what you need, full list of commands/options/properties that can be controlled are at https://mpv.io/manual/master/

**Reply by 1000Fear** [reply]
Thanks a lot :)

**Comment by hooke007** [comment, marked as answer]
Learn how to use cli on WIN.
https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/powershell

And, are you unsatisfied with the manual? What else cmds would you need...
https://mpv.io/manual/master/#list-of-input-commands

**Reply by 1000Fear** [reply]
Thank you for your guidance! It seems that I still don't have enough understanding of the manual :)
